solidity/contracts/libs/CheckpointLib.sol

```diff
@@ -24,7 +25,8 @@ library CheckpointLib {
                     abi.encodePacked(
                         _domainHash,
                         _checkpointRoot,
-                        _checkpointIndex
+                        _checkpointIndex,
```

Comment:
Is it safe to use the same domain separator? I think so, but want to double check...

Reply:
discussed offline, this is safe